Plugins/Json/src/MaterialMapJsonConverter.cpp

```diff
@@ -28,6 +28,44 @@
 #include <algorithm>
 #include <map>
 
+namespace Acts {
+// specialisations of decoration helper function
+// to pick correct objects from the contaier object
+template <>
+inline void decorateJson<Acts::SurfaceAndMaterialWithContext>(
+    const ITrackingGeometryJsonDecorator* decorator,
+    const Acts::SurfaceAndMaterialWithContext& src, nlohmann::json& dest) {
+  if (decorator != nullptr && std::get<0>(src)) {
```

Comment:
Despite the line below, I don't think it's immediately obvious that ```std::get<0>(src)``` returns a pointer, so for readability I think it would help if it is explicitly checked against ```nullptr```, too

Reply:
done. Thanks!